src/_posts/2014-02-19-appveyor-20-dedicated-build-vms-parallel-testing-nuget-deployment.md

```diff
@@ -23,7 +23,7 @@ deployment.
 * [NuGet hosting](#nuget)
 * [Build matrix](#build-matrix)
 * [Parallel testing](#parallel-testing)
-* [Deployment to multiple environments](#deployment)
+* [Deployment to multiple environments](#deploying)
```

Comment:
Why this change? I mean, it will break old references which we can't control ourselves.